Exercise_D/assets/style/index.css

```diff
@@ -0,0 +1,76 @@
+/* 
+This css sheet is linked to practice.html
+You can style tags, classes (.) or ids (#) from this file.
+*/
+
+.head{
+    background:linear-gradient(rgba(0,0,0,0.5), rgba(0,0,0,0.5)), url(/Exercise_D/assets/images/flowershop.jpg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    display: flex;
+    width: 100%;
+    height: 80vh;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+}
+
+.heading{
+    letter-spacing: 3px;
+}
+
+.whitetxt{
+    color: #fff;
+}
+
+.large{
+    font-size: xx-large;
+}
+
+.medium{
+    font-size: x-large;
+}
+
+.green{
+    border-radius: 20px 20px 20px 20px;
```

Comment:
border-radius: 20px should apply it all around. 
Try to stick to one colour convention when working on a CSS sheet.